hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -43,89 +18,92 @@ This section describes all the configuration items available in the bootloader.
 
 If 1 then enable UART debug output on GPIO 14 and 15. Configure the receiving debug terminal at 115200bps, 8 bits, no parity bits, 1 stop bit.
 
-Default: 0  
+Default: `0`  
 Version: All  
 
 ### WAKE_ON_GPIO
 
 If 1 then 'sudo halt' will run in a lower power mode until either GPIO3 or GLOBAL_EN are shorted to ground.
 
-Default: 1 (0 in original version of bootloader 2019-05-10)  
+Default: `1` (`0` in original version of bootloader 2019-05-10)  
 Version: All  
 
 ### POWER_OFF_ON_HALT
 
 If 1 and WAKE_ON_GPIO=0 then switch off all PMIC outputs in halt. This is lowest possible power state for halt but may cause problems with some HATs because 5V will still be on. GLOBAL_EN must be shorted to ground to boot.
```

Comment:
`1` in backticks.

Reply:
Also might want some further explanation of "halt", e.g.:

`If 1 and WAKE_ON_GPIO=0 then switch off all PMIC outputs when the Pi is shut down.`

And `WAKE_ON_GPIO=0` in backticks.

Reply:
reworded